.github/actions/create-featurenet/action.yml

```diff
@@ -129,7 +141,10 @@ runs:
         pr_image="${{ inputs.ecr-public-registry }}feature-env-aleph-node:${pr_image_tag}"
         ./Ops.sh create-featurenet \
           "${{ env.APP_NAME }}" \
-          "${pr_image}"
+          "${pr_image}" \
+          "false" \
+          "false" \
+          "${{ inputs.update == 'true' && inputs.rolling-update-partition || '0' }}"
```

Comment:
We have in total three added args, but Ops.sh will have in total added 2?

Reply:
That is not going to cause any problems.  There is an another PR in another repository to update that Ops.sh script where partition is actually last argument.  In that future version of Ops.sh the `create-featurenet` command goes as follows:

    create-featurenet <feature-net-name> <aleph-node-image|'testnet'|'mainnet'> [create-hook='false'] [generate-data-in-s3='false'] [rolling-update-partition='0']"